### PR TITLE
workflows: release to homebrew

### PR DESCRIPTION
**Pull request by chenrui333**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Follow up of #659, a repo token and homebrew-core would be needed.

cc @schosterbarak @nimrodkor

### CODE REVIEW

File: .github/workflows/homebrew.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  homebrew:
+    name: Bump Homebrew formula
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+          formula-name: checkov
+        env:
+          COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
```
Comment (nimrodkor) [inline .github/workflows/homebrew.yml:17]: Thanks @chenrui333 !

We do have a token already in our secrets, can we reuse it?
It's in use in: https://github.com/bridgecrewio/checkov/blob/52c7a73f9788a88a7f90408309c5fb6fa827f4d5/.github/workflows/build.yml#L176
```suggestion
          COMMITTER_TOKEN: ${{ secrets.PAT_TOKEN }}
```

Comment (chenrui333) [reply .github/workflows/homebrew.yml:17]: Nice!! that is what I want.

**Review by nimrodkor — APPROVED** [review]

File: .github/workflows/homebrew.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: homebrew
+
+on:
+  push:
+    tags: 'v*'
```
Comment (chenrui333) [inline .github/workflows/homebrew.yml:5]: this needs to be `*` instead of `v*`

Comment (chenrui333) [reply .github/workflows/homebrew.yml:5]: I just noticed the tags do not have `v`

**Pull request merged**